docs/source/concepts/AbsorptionAndMultipleScattering_plot_common.py

```diff
@@ -75,9 +76,9 @@ def get_montecarlo_results(sample_ws):
     Y=[1,1,1,1,1]
     SQ=CreateWorkspace(DataX=X,DataY=Y,UnitX="MomentumTransfer")
     ConvertUnits(InputWorkspace=sample_ws, OutputWorkspace=sample_ws, Target="Momentum")
-    mc_corrections = DiscusMultipleScatteringCorrection(sample_ws, StructureFactorWorkspace=SQ, NumberScatterings=2)
+    DiscusMultipleScatteringCorrection(sample_ws, StructureFactorWorkspace=SQ, NumberScatterings=2, OutputWorkspace="mc_corrections")
     # apply ms correction using ratio method
-    multi_scatt_ratio = mc_corrections[1]/(mc_corrections[1] + mc_corrections[2])
+    multi_scatt_ratio = mtd["mc_corrections_Scatter_1"]/(mtd["mc_corrections_Scatter_1"] + mtd["mc_corrections_Scatter_2"])
```

Comment:
Sorry I have to un-assign myself as realised I can't build docs on non-conda...sorry!

Btw if you still returned the workspace group from `DiscusMultipleScatteringCorrection` I think it would be possible to have
`multi_scatt_ratio =  mc_corrections[1]/(mc_corrections[1] + mc_corrections[3])`
but I suppose using the names makes it easier to interpret and also allow you to add more workspaces to the group at some point, is that the thinking?